src/pages/ExpiringResults/index.js

```diff
@@ -0,0 +1,295 @@
+import React, { Component } from 'react';
+import { connect } from 'dva';
+import {
+  Grid,
+  GridItem,
+  Card,
+  TextContent,
+  Text,
+  TextVariants,
+  Button,
+  Progress,
+  ProgressSize,
+  ProgressMeasureLocation,
+  ProgressVariant,
+  Tooltip,
+} from '@patternfly/react-core';
+import { Icon } from 'antd';
+import { EllipsisVIcon } from '@patternfly/react-icons';
+import { formatDate, getDiffDays, getDiffDate } from '../../utils/moment_constants';
+import Table from '@/components/Table';
+import styles from './index.less';
+
+const expirationLimit = { val: 30 };
```

Comment:
Does this need to be an object or can we simply assigned it to 30? Additionally, constants should be all uppercase (e.g. `EXPIRATION_LIMIT`) for improved readability.

Reply:
We need to discuss whether the value is constant or not and if not, where is the value stored. @portante @dbutenhof

Reply:
This is an attribute we're going to want to maintain on the server. In fact I suspect we'll have a tier of settings, with server defaults (and possibly maximums) for private and shared values, plus override values for each user. The latter is probably user metadata, which will be enabled by Nikhil's #2049; I'm not quite sure how we maintain the server defaults and limits. We probably need a new configuration API to get/set pre-defined values that aren't associated with any user or dataset, and we haven't talked about those. (Alternately, I suppose, these could be `pbench-server.cfg` values, although I like the idea of opening some class of configuration settings to more dynamic management through a hypothetical future dashboard "server management" page.)

For now, a hardcoded dashboard value is probably fine, but let's not treat it as a "constant" because in the long run, it won't be. Creating a redux store for "server configuration" is probably a good idea, and it can for now be hardcoded there until we have the server APIs built.